### PR TITLE
refactor(ark-survey): renamed methods, use queryBus, remove unused imports, camelCase date fields

### PR DESCRIPTION
**Pull request by bennyvdhoogen**
BREAKING CHANGE: removed deleted_at column in arkSurveyReachSegments, renamed local date variables in Ark models to camelCase.

Prisma schema changes:

removed deleted_at in arkSurveyReachSegments
updated precision on float fields in arkSurvey (apparently this was not reflected in the schema yet, not sure if that affects anything).

Depending on migrations from MR in AIP repo: https://git.data.amsterdam.nl/bmi/aip/-/merge_requests/1537

### CODE REVIEW

File: src/schema/ark-survey/ark-survey.resolver.ts [hunk 1]
```diff
@@ -63,10 +62,10 @@ export class ArkSurveyResolver {
 		return ArkSurveyFactory.createArkSurvey(domainArkSurvey);
 	}
 
-	@Query((returns) => ArkSurvey, { name: 'getArkSurvey' })
+	@Query(() => ArkSurvey)
 	@Roles({ roles: ['realm:aip_owner', 'realm:aip_admin', 'realm:aip_survey'], mode: RoleMatchingMode.ANY })
-	async getArkSurvey(@Args('surveyId', { type: () => String }) surveyId: string) {
-		return this.arkSurveyService.getArkSurveyData(surveyId);
+	public async getArkSurvey(@Args('surveyId') surveyId: string): Promise<ArkSurvey> {
+		return this.queryBus.execute<GetArkSurveyBySurveyIdQuery>(new GetArkSurveyBySurveyIdQuery(surveyId));
```
Comment (fortunato) [inline src/schema/ark-survey/ark-survey.resolver.ts:68]: Nice one. Any chance we can the convert the command on the field resolver below to a query as well?

**Review by fortunato — CHANGES_REQUESTED** [review]
I spotted a few useless assignments. Can you take a look at those?
Can we remove the `hasReachSegments` and `CreateReachSegmentsFromJson` methods? They appear unused.
As discussed on slack, let's merge the ark-survey and reach-segment modules as are tightly coupled.

Looks neat overall.

File: src/schema/ark-survey/ark-survey.resolver.ts (hunk 1 above)
Comment (bennyvdhoogen) [reply src/schema/ark-survey/ark-survey.resolver.ts:67]: Sure thing, done!

**Comment by bennyvdhoogen** [comment]
Related to JIRA ticket: https://gemeente-amsterdam.atlassian.net/browse/BAIP-188

**Review by fortunato — APPROVED** [review]

**Pull request merged**